sample-apps/celery_sample/README.md

```diff
@@ -0,0 +1,37 @@
+# Django Sample App
```

Comment:
The README needs updated for how to setup and run the Celery sample app.

Reply:
I will updated today.